src/components/edit/BoundaryDialog.js

```diff
@@ -118,11 +118,8 @@ class BoundaryDialog extends Component {
         return (
             <div data-test="boundarydialog">
                 <Tabs value={tab} onChange={tab => this.setState({ tab })}>
```

Comment:
I wonder if this can be simplified by using directly `TabBar` and `Tab` from `ui` in these places.
The custom components `Tabs` and `Tab` don't seem to do anything special that won't work with the `ui` counterparts?
Maybe wrapping was necessary before because of MUI? Or was it a choice to wrap everything to not bind Maps app to any particular UI library and allow for easy replacement of the underlying UI lib?

Reply:
TabBar don't have an onChange handler, so each tab needs a selected state and an onClick handler, so I felt this was simplifying the code on the layer dialogs.